mono/metadata/console-unix.c

```diff
@@ -216,50 +216,24 @@ tty_teardown (void)
 	setup_finished = FALSE;
 }
 
+static GENERATE_TRY_GET_CLASS_WITH_CACHE(system_console, "System", "Console")
```

Comment:
This should generate a static function `mono_class_try_get_system_console_class` which should be ok to call repeatedly (it should store a static copy of the `MonoClass*` internally).

So if you do that and use the method you shouldn't need to add a `MonoClass:console_class` field.

Reply:
That's a leftover, and should be removed

Reply:
Ok, that works too.  Is there any particular reason to prefer a larger MonoDefaults over a caching getter?